### PR TITLE
http-json-graphite

### PR DESCRIPTION
**Pull request by paulhamby**
Hits an HTTP endpoint which emits JSON and pushes data into Graphite.

### CODE REVIEW

**Comment by mattyjones** [comment]
@paulhamby Can you do a rebase, and then I should be able to merge this.

Thanks

**Comment by analytically** [comment]
Rebase?

File: bin/http-json-graphite.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+#   graph. The object option is optional and is the name of the JSON object
+#   which holds the key/value pairs you want to graph.
+#
+# LICENSE:
+#   phamby@gmail.com
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
```
Comment (eheydrick) [inline bin/http-json-graphite.rb]: this isn't needed since the plugin is only supported on Ruby 1.9.3+

**Comment by paulhamby** [comment]
My apologies for being unresponsive on this PR. What should the new name of the plugin be?

Thanks!
Paul

**Comment by eheydrick** [comment]
I suggest metrics-http-json.rb It's not really specific to graphite, right?

**Comment by caskolkm** [comment]
any idea when this PR will be merged?

**Comment by paulhamby** [comment]
Opened a new PR: https://github.com/sensu-plugins/sensu-plugins-http/pull/26

**Pull request closed without merging**